### DISCUSSION TITLE
In remote desktop image, the upload function does not work.

### DISCUSSION THREAD

**Discussion by wycc** [wycc/cgu-kubeflow#5] [General]
This is becasue that the tiny file manager use some absolute path. However, we convert the http to https in by using a nginx reverse proxy. The result is that we send a http request in a https link. Therefore, the browser block this access for security issue.

We should replace the index.php of the tiny file manager inside the remote desktop image.
![image](https://github.com/wycc/cgu-kubeflow/assets/457473/a954e4ac-4d7b-455b-97ee-cc24df610466)

The above change will fix the issue. Please repleace the index.php in the remotedesktop image and build the docker image for both remote-desktop, remote-desktop-ros and remote-desktop-ros-eng.

**Comment by WinterBelieve** [comment]
Where are we going to find this file?

**Comment by wycc** [comment]
![image](https://github.com/wycc/cgu-kubeflow/assets/457473/2a769523-756e-4b21-80db-5b05a486aeaa)
 The tinyfilemanager.php will be copied to the /var/www/html/index.html. We can change the tinyfilemanager.php.

**Comment by Liuvert** [comment]
The copying is successful. However, we have some problems.

**Comment by WinterBelieve** [comment]
<img width="478" alt="image" src="https://github.com/wycc/cgu-kubeflow/assets/61958761/8d6648ee-4ea9-485a-9759-da53dcb36da1">

Is it okay if we don't need tinyfilemanager.php as the homepage but instead have it as a resource accessible via a specific URL?
